collection-scripts/gather_ceph_resources

```diff
@@ -374,7 +374,20 @@ for ns in $namespaces; do
                     timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool info $bp --format=json --pretty-format" 2>>"${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-pool-info-"${bp}"-debug.log
                 } >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_pool_info &
                 pids_rbd+=($!)
-                printf "Collecting mirror image status for images in: %s\n" "${bp}" >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_image_status
+                # Collecting mirror image status for provider mode with rados namespace
+                rados_namespaces=$(timeout 60 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd -p $bp namespace ls") 
+                for rns in $rados_namespaces; do
+                    printf "Collecting mirror image status for images in: %s/%s\n" "${bp}" "${rns}" >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_image_status
```

Comment:
Use `dbglog` please (it also appends to the log file)

Reply:
We are already appending one below for individual images, I don't think it is required here. Also, we are doing same for collection of other resources. If required we can send a cleanup PR for this.